webdriver/tests/bidi/browsing_context/capture_screenshot/invalid.py

```diff
@@ -21,41 +21,40 @@ async def test_invalid_frame(bidi_session, value):
         await bidi_session.browsing_context.capture_screenshot(context=value)
 
 
-async def test_closed_frame(bidi_session, top_context, inline, add_and_remove_iframe):
+async def test_closed_frame(bidi_session, top_context, inline,
```

Comment:
Mind reverting the formatting changes to make it easier to see the actual important changes? Thanks.